<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>syslog-ng config</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="fixedWidth">
       <number>165</number>
      </property>
      <widget class="QWidget" name="sourceTab">
       <attribute name="title">
        <string>Source</string>
       </attribute>
       <layout class="QVBoxLayout" name="sourceMainLayout">
        <item>
         <widget class="QScrollArea" name="sourceScrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Tab" name="sourceWidget">
           <layout class="QVBoxLayout" name="sourceLayout">
            <property name="Alignment">
             <enum>Qt::AlignTop</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="destinationTab">
       <attribute name="title">
        <string>Destination</string>
       </attribute>
       <layout class="QVBoxLayout" name="destinationMainLayout">
        <item>
         <widget class="QScrollArea" name="destinationScrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Tab" name="destinationWidget">
           <layout class="QVBoxLayout" name="destinationLayout">
            <property name="Alignment">
             <enum>Qt::AlignTop</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filterTab">
       <attribute name="title">
        <string>Filter</string>
       </attribute>
       <layout class="QVBoxLayout" name="filterMainLayout">
        <item>
         <widget class="QScrollArea" name="filterScrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Tab" name="filterWidget">
           <layout class="QVBoxLayout" name="filterLayout">
            <property name="Alignment">
             <enum>Qt::AlignTop</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="templateTab">
       <attribute name="title">
        <string>Template</string>
       </attribute>
       <layout class="QVBoxLayout" name="templateMainLayout">
        <item>
         <widget class="QScrollArea" name="templateScrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Tab" name="templateWidget">
           <layout class="QVBoxLayout" name="templateLayout">
            <property name="Alignment">
             <enum>Qt::AlignTop</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rewriteTab">
       <attribute name="title">
        <string>Rewrite</string>
       </attribute>
       <layout class="QVBoxLayout" name="rewriteMainLayout">
        <item>
         <widget class="QScrollArea" name="rewriteScrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Tab" name="rewriteWidget">
           <layout class="QVBoxLayout" name="rewriteLayout">
            <property name="Alignment">
             <enum>Qt::AlignTop</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="parserTab">
       <attribute name="title">
        <string>Parser</string>
       </attribute>
       <layout class="QVBoxLayout" name="parserMainLayout">
        <item>
         <widget class="QScrollArea" name="parserScrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="Tab" name="parserWidget">
           <layout class="QVBoxLayout" name="parserLayout">
            <property name="Alignment">
             <enum>Qt::AlignTop</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="sceneWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="sceneMainLayout">
       <item>
        <widget class="QScrollArea" name="sceneScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <!--<widget class="QStatusBar" name="statusBar"/>-->
  <!--<widget class="QToolBar" name="toolBar"/>-->
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>New &amp;log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Global options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tab</class>
   <extends>QWidget</extends>
   <header>tab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
